src/CMakeLists.txt

```diff
@@ -23,3 +23,6 @@ target_include_directories(memory_pool PRIVATE ${CMAKE_CURRENT_LIST_DIR})
 
 # Add compile options for the memory pool target
 target_compile_options(memory_pool PRIVATE -Wall -Wextra -Werror -Wno-format -g)
+
+add_custom_target(update_timestamp COMMAND ${CMAKE_COMMAND} -E touch
```

Comment:
The selected line adds a custom target named `update_timestamp` to the CMake build system. This target executes the `touch` command on the malloc.c file located in the current list directory, which updates its timestamp. This can be useful to **force** recompilation of the malloc.c file.